README.md

```diff
@@ -8,3 +8,29 @@ An implementation of Circonus log-linear histograms.
     ./configure
     make
     make install
+
+## Tests
+
+    make tests
+
+## Documentation
+
+The circllhist API is documented in the header file [circllhist.h](src/circllhist.h).
+A html version can be generated using:
+
+    make docs
+
+This requires doxygen to be available on your system.
+
+## Python bindings
+
+Build and install the C library as described above.
+Then use the following command to install the python library.
+
+    cd src/python
+    python setup.py install
+
+## Lua bindings
+
+This repository contains lua bindings for libcircllhist, which are generated my `make`,
```

Comment:
Typos:

"which are generated **by** `make`"

"and will be installed **by** `make install`"